test/cmd/newapp.sh

```diff
@@ -62,6 +62,8 @@ os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'MY
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_USERNAME'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_PASSWORD'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample --param MYSQL_PASSWORD=hello -o yaml' 'hello'
+os::cmd::expect_success_and_text  'oc new-app -e FOO=BAR -f examples/jenkins/jenkins-ephemeral-template.json -o json | grep FOO | wc -l' '1'
+os::cmd::expect_success_and_text  'oc new-app -e OPENSHIFT_ENABLE_OAUTH=false -f examples/jenkins/jenkins-ephemeral-template.json -o json | grep -A 1 OPENSHIFT_ENABLE_OAUTH | grep false | wc -l' '1'
```

Comment:
You'll need to surround the JSONPath with braces:
```
oc get dc/jenkins -o jsonpath='{.spec.replicas}'
```

See the [k8s documentation](http://kubernetes.io/docs/user-guide/jsonpath/) for more detail.